AUTHORS

```diff
@@ -17,5 +17,7 @@ Shivam Vats <shivamvats.iitkgp@gmail.com>
 Sumith <sumith1896@gmail.com>
 Sanka Rasnayaka <nuwan.ras@gmail.com>
 Abinash Meher <abinashdakshana999@gmail.com>
-AMiT Kumar <dtu.amit@gmail.com>
+Govind Sahai <gsiitbhu@gmail.com>
+Amit Kumar <dtu.amit@gmail.com>
```

Comment:
I am using the capitalised version `AMiT Kumar` now, so this needs to be added to mailmap as well, otherwise it's good +1